apps/fabric-website/src/pages/HomePage/HomePage.base.tsx

```diff
@@ -0,0 +1,264 @@
+          <div className={classNames.oneFourth}>
+            <p>
+              We're broadening our guidance to include more platforms and create an open source system, making it possible for us all to
+              evolve together.
+            </p>
+          </div>
+        </div>
+      </section>
+    );
+  };
+
+  private _renderResourcesSection = (): JSX.Element => {
+    return (
+      <section className={this._classNames.resourcesSection}>
+        <div className={this._classNames.sectionContent}>
+          <div className={this._classNames.oneHalf}>
+            <Image
+              src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/discover-resources-1x.png"
+              alt="Resources illustration"
+              className={this._classNames.illustration}
+            />
+          </div>
+          <div className={this._classNames.oneFourth}>
+            <h2 className={this._classNames.resourcesTitle}>Discover resources</h2>
+            <p>Find design, inclusive and developer onboarding resources, and learn about how to become a contributor.</p>
+            <p>{this._renderLink('#/resources', 'See resources')}</p>
+          </div>
+        </div>
+      </section>
+    );
+  };
+
+  private _renderUsageSection = (): JSX.Element => {
+    return (
+      <section className={this._classNames.usageSection}>
+        <div className={this._classNames.sectionContent}>
+          <div className={this._classNames.oneFourth}>
+            <h2 className={this._classNames.usageTitle}>Who at Microsoft uses Fabric?</h2>
+            <p>From Word, PowerPoint and Excel to PowerBI, many teams in Microsoft utilize the functionality of Fabric.</p>
+          </div>
+          <div className={this._classNames.oneFourth} />
+          <figure className={this._classNames.oneHalf}>
+            <ul className={this._classNames.usageIconList}>{this._renderUsageIconList()}</ul>
+            <figcaption>
+              <strong>+many additional Microsoft sites and products</strong>
+            </figcaption>
+          </figure>
+        </div>
+      </section>
+    );
+  };
+
+  /**Renders a link with an icon */
+  private _renderLink = (url: string, text: React.ReactNode, disabled?: boolean, isCTA?: boolean, icon = 'Forward'): JSX.Element => {
+    return (
+      <Link
+        className={this._classNames.link}
+        href={url}
+        disabled={!!disabled}
+        // tslint:disable-next-line jsx-no-lambda
+        onClick={ev => (isCTA ? this._onCTAClick(ev) : this._onInternalLinkClick(ev, url))}
+      >
+        <Icon iconName={icon} className={this._classNames.linkIcon} />
+        <span className={this._classNames.linkText}>{text}</span>
+      </Link>
+    );
+  };
+
+  /** Renders list of app/brand icons that use Fabric. */
+  private _renderUsageIconList = (): JSX.Element[] => {
+    return fabricUsageIcons.map((icon, iconIndex) => (
+      <li key={iconIndex} className={this._classNames.usageIconListItem}>
```

Comment:
It's generally considered best practice to avoid using indices as keys whenever possible. If the item being rendered has any property which is unique within the list of items, it's better to use that instead. (Search for "index" [here](https://reactjs.org/docs/lists-and-keys.html) for background.) Though practically speaking it's not super important in this case because the list of items will be rendered once and not change.